agent/app/agent.go

```diff
@@ -488,18 +491,19 @@ func (agent *ecsAgent) newTaskEngine(containerChangeEventStream *eventstream.Eve
 	credentialsManager credentials.Manager,
 	state dockerstate.TaskEngineState,
 	imageManager engine.ImageManager,
-	execCmdMgr execcmd.Manager) (engine.TaskEngine, string, error) {
+	execCmdMgr execcmd.Manager,
+	appnetAgentloader serviceconnect.Loader) (engine.TaskEngine, string, error) {
```

Comment:
ditto the same naming nit here, seems like var name should match the package name (serviceConnectLoader)